src/StorageSync/StorageSync/StorageSync.csproj

```diff
@@ -11,6 +11,8 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Reference Include="System.Security" />
+    <PackageReference Include="Microsoft.Azure.Management.Authorization" Version="2.13.0-preview" />
```

Comment:
please try Microsoft.Azure.Management.Authorization.Version2015_07_01 instead

Reply:
I tried 
- Container differences of direct properties vs "Properties" nested container
- ListForResource have one extra parameter parentResourcePath , which I couldnt understand. Also, ResourceIdentitifier is not available latest.

We are only one version short with this package, Given deadline, We will remove this package in next release unless you are aware of some critical defect in this package.

Reply:
after discussion let's go ahead with the package in csproj, will try to fix the SDK afterwards